net-gateway/src/endpoints/files/pcap_file.rs

```diff
@@ -0,0 +1,98 @@
+use actix_web::post;
+use actix_web::web;
+use actix_web::HttpRequest;
+use actix_multipart::Multipart;
+
+use futures::StreamExt;
+use futures::TryStreamExt;
+
+use net_core_api::api::envelope::envelope::Envelope;
+use net_core_api::api::result::result::ResultDTO;
+use net_core_api::core::decoder_api::Decoder;
+use net_core_api::core::typed_api::Typed;
+use net_core_api::core::encoder_api::Encoder;
+
+use net_inserter_api::api::pcap_file::InsertPcapFileDTO;
+use net_token_verifier::fusion_auth::fusion_auth_verifier::FusionAuthVerifier;
+
+use crate::core::quinn_client_endpoint_manager::QuinnClientEndpointManager;
+use crate::core::user_facing_error::UserFacingError;
+use crate::{authorization, config::Config};
+
+#[post("/pcap-file")]
```

Comment:
Still voting for `insert` in the name somewhere. It is impossible to understand, what is going on if not diving into the code

Reply:
`We should not use URIs to indicate a CRUD function. URIs should only be used to identify the resources and not any action upon them uniquely.`. You should read it like post pcap-file